src/apify_client/clients/resource_clients/webhook.py

```diff
@@ -0,0 +1,130 @@
+from typing import Any, Dict, List, Optional
+
+from ..base.resource_client import ResourceClient
+from .webhook_dispatch_collection import WebhookDispatchCollectionClient
+
+
+def prepare_webhook_representation(
+    event_types: Optional[List] = None,
```

Comment:
lets have `*` args as the first arg in this list to force user to user named params.

Reply:
Users shouldn't use this function at all.

Reply:
yes, but it will force us to use named args, which is good practice :-)